.ci/pipeline_definitions

```diff
@@ -42,13 +42,13 @@ etcd-backup-restore:
                 build: ~
     steps:
       check:
-        image: 'golang:1.17'
+        image: 'golang:1.18'
```

Comment:
Can you update pipeline to use `1.18.6` as druid pipeline is also at `1.1.8.5`

Reply:
Updated